rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -681,6 +681,58 @@ private Statement labeled(org.codehaus.groovy.ast.stmt.Statement statement, Supp
             return s;
         }
 
+        @Override
+        public void visitArrayExpression(ArrayExpression expression) {
+            Space whitespaceBeforeNew = whitespace();
```

Comment:
```suggestion
            Space prefix = whitespace();
```

Reply:
No problem, consistency is important! Thanks for the correction.